src/NzbDrone.Core/Notifications/Webhook/WebhookSettings.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using FluentValidation;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+using NzbDrone.Core.Validation.Paths;
+
+namespace NzbDrone.Core.Notifications.Webhook
+{
+    public class WebhookSettingsValidator : AbstractValidator<WebhookSettings>
+    {
+        public WebhookSettingsValidator()
+        {
+            RuleFor(c => c.Url).IsValidUrl();
+        }
+    }
+
+    public class WebhookSettings : IProviderConfig
+    {
+        private static readonly WebhookSettingsValidator Validator = new WebhookSettingsValidator();
+
+        [FieldDefinition(0, Label = "URL", Type = FieldType.Url)]
+        public String Url { get; set; }
```

Comment:
nit: `string` instead of `String` (things are a bit all over with this one, but we've landed on using `string`).